ppr-ui/src/components/mhrRegistration/ReviewConfirm/YourHomeReview.vue

```diff
@@ -179,6 +179,16 @@ export default defineComponent({
       }),
       engineerDisplayDate: computed((): string => {
         return yyyyMmDdToPacificDate(getMhrRegistrationHomeDescription.value?.engineerReportDate, true)
+      }),
+      manufactureYear: computed((): string => {
+        const year = getMhrRegistrationHomeDescription.value.baseInformation.year
+        if (!year) {
+          return '(Not Entered)'
+        }
+        if (getMhrRegistrationHomeDescription.value.baseInformation.circa) {
+          return 'Circa ' + year
+        }
+        return year
```

Comment:
Are Base Information / circa / year always initialized before this is processed? 
If not, might want to protect yourself from type errors here by optional chaining. 

These would show up in the console if there was an issue 👍

Reply:
the model was initialized like this
![image](https://user-images.githubusercontent.com/62360407/181110821-76c27a6e-c662-4fe7-8356-42d68235d767.png)
so I think it should be ok?

Reply:
Should be fine, as long as you don't see any errors in the console when firing up the Registration 👍